helm_deploy/values-preprod.yaml

```diff
@@ -0,0 +1,16 @@
+---
+# Per environment values which override defaults in send-legal-mail-to-prisons-api/values.yaml
+
+generic-service:
+  replicaCount: 2
+
+  ingress:
+    host: send-legal-mail-api-preprod.prison.service.justice.gov.uk
+
+  env:
+    APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.json
```

Comment:
You don't want an `applicationinsights.preprod.json`? Probably not

Reply:
Good question - I haven't seen one on any other projects, I guess we want preprod to be as prod like as possible.